hw7/functional.py

```diff
@@ -0,0 +1,58 @@
+def sequential_map(*args):
+    container = list(args[-1])
+    functions = args[0:(len(args)-1)]
+    for function in functions:
+        container = list(map(function, container))
+    return container
```

Comment:
Превратить элемент в список можно только в самом конце, map можно передавать в другие map'ы
```suggestion
    for function in functions:
        container = map(function, container)
    return list(container)
```